conversion/conversion.go

```diff
@@ -429,13 +429,8 @@ func CreateDatabase(project, instance, dbName string, conv *internal.Conv, out *
 
 // UpdateDDLForeignKeys updates the Spanner database with foreign key
 // constraints using ALTER TABLE statements.
-func UpdateDDLForeignKeys(project, instance, dbName string, conv *internal.Conv, out *os.File) error {
+func UpdateDDLForeignKeys(project, instance, dbName string, maxWorkers int64, conv *internal.Conv, out *os.File) error {
 	ctx := context.Background()
-	adminClient, err := database.NewDatabaseAdminClient(ctx)
-	if err != nil {
-		return fmt.Errorf("can't create admin client: %w\n", analyzeError(err, project, instance))
-	}
-	defer adminClient.Close()
```

Comment:
My understanding is that you don't a separate adminClient for each of the parallel go threads - see https://github.com/googleapis/google-cloud-go/blob/691e923fca9bd3194ff4ba49bd2d899518875d7c/spanner/admin/database/apiv1/database_admin_client.go#L261 for discussion on it being concurrency safe.

Reply:
Interesting let me change that